index.html

```diff
@@ -950,10 +950,10 @@ <h3>Fundraising events organiser</h3>
 					</div>
 
 
-
-					<h3>Travel</h3>
+					
+					<h3>Travel (77 countries overall)</h3>

```

Comment:
I think this should be between 45 and 50 thereabouts

Reply:
you might know this already but THIS is what maths actually is - deciding WHAT to do, not the actual maths itself :slightly_smiling_face: So this is good practice!